tests/testMLTflareModel.py

```diff
@@ -0,0 +1,731 @@
+                       '{"m": "MLT",
+                         "p": {"lc": "lc_1.txt", "t0": 456.2}}',
+                       0.25, 2.432, 17.1)''')
+
+        cursor.execute('''INSERT INTO mlt_test VALUES( 1, 25.2, 32.0,
+                       'lte028-5.0+0.5a+0.0.BT-Settl.spec.gz',
+                       '{"m": "MLT",
+                         "p": {"lc": "lc_1.txt", "t0": 41006.2}}',
+                       0.15, 1.876, 17.2)''')
+
+        cursor.execute('''INSERT INTO mlt_test VALUES( 2, 25.3, 10.0,
+                       'lte028-5.0+0.5a+0.0.BT-Settl.spec.gz',
+                       '{"m": "MLT",
+                         "p": {"lc": "lc_2.txt", "t0": 117.2}}',
+                       0.3, 2.654, 17.3)''')
+
+        cursor.execute('''INSERT INTO mlt_test VALUES( 3, 25.4, 11.0,
+                       'lte028-5.0+0.5a+0.0.BT-Settl.spec.gz',
+                       '{"m": "MLT",
+                         "p": {"lc": "lc_2.txt", "t0": 10456.2}}',
+                       0.22, 2.364, 17.4)''')
+        conn.commit()
+        conn.close()
+
+
+    @classmethod
+    def tearDownClass(cls):
+        if os.path.exists(cls.mlt_lc_name):
+            os.unlink(cls.mlt_lc_name)
+
+        if os.path.exists(cls.db_name):
+            os.unlink(cls.db_name)
+
+    def test_flare_lc_failure(self):
+        """
+        Test that the correct exception is thrown when you try
+        to interpolate from an MLT light curve cache that does not
+        exist
+        """
+        cat_name = os.path.join(getPackageDir('sims_catUtils'),
+                                'tests', 'scratchSpace',
+                                'flaring_cat_failure.txt')
+        db = MLT_test_DB(database=self.db_name, driver='sqlite')
+        obs = ObservationMetaData(mjd=67432.1)
+        flare_cat = FlaringCatalog(db, obs_metadata=obs)
+        flare_cat._mlt_lc_file = 'a_nonexistent_cache'
+        with self.assertRaises(RuntimeError) as context:
+            flare_cat.write_catalog(cat_name)
+        self.assertIn('get_mdwarf_flares.sh',
+                      context.exception.args[0])
+
+        if os.path.exists(cat_name):
+            os.unlink(cat_name)
+
+    def test_flare_magnitudes(self):
+        """
+        Test that we get the expected magnitudes out
+        """
+        db = MLT_test_DB(database=self.db_name, driver='sqlite')
+
+        # load the quiescent SEDs of the objects in our catalog
+        sed_list = SedList(['lte028-5.0+0.5a+0.0.BT-Settl.spec.gz']*4,
+                           [17.1, 17.2, 17.3, 17.4],
+                           galacticAvList = [2.432, 1.876, 2.654, 2.364])
+
+        bp_dict = BandpassDict.loadTotalBandpassesFromFiles()
+
+        # calculate the quiescent fluxes of the objects in our catalog
+        baseline_fluxes = bp_dict.fluxListForSedList(sed_list)
+
+        bb_wavelen = np.arange(100.0, 1600.0, 0.1)
+        bb_flambda = blackbody_lambda(bb_wavelen*10.0, 9000.0)
```

Comment:
Nice. I was going to suggest this. Glad you used an outside version as a test. If these are passing I guess my earlier comment about the form of the blackbody must be wrong.